playground/mocks/spec-okta-api/oauth2/v1/authorize/callback.js

```diff
@@ -0,0 +1,22 @@
+const apis = [
```

Comment:
why to put this under `v1` code ?

Reply:
We do not have an `/oauth2/v2` endpoint. This flow is disjoint from our IDX remediation flows.